go.mod

```diff
@@ -2,7 +2,7 @@ module github.com/netfoundry/ziti-sdk-golang
 
 go 1.13
 
-// replace github.com/netfoundry/ziti-foundation => ../ziti-foundation
+replace github.com/netfoundry/ziti-foundation => ../ziti-foundation
```

Comment:
i'll have to fix this - i thought i was up to date but i seem to have missed some spots :(